src/components/CertificateDropZone/Views/UnverifiedView.js

```diff
@@ -13,7 +13,7 @@ const DetailedErrors = ({ verificationStatus }) => {
   if (!get(verificationStatus, "identity.identifiedOnAll"))
     errors.push(TYPES.IDENTITY);
   const renderedError = errors.map((errorType, index) => (
-    <div key={index}>
+    <div id={`error-tab${index}`} key={index}>
```

Comment:
I suggest to use errorType instead of index => `error-tab-${errorType}`. it's more description than tab0, tab1, ..

Reply:
moved id to parent div to test multiple error case easily.